### PR TITLE
sdl/all: bump deps

### PR DESCRIPTION
**Pull request by ericLemanissier**
Generated and committed by [Conan Center Bump Deps](https://github.com/ericLemanissier/conan-center-index-bump-deps) Find more updatable recipes in the [GitHub Pages](https://ericlemanissier.github.io/conan-center-index-bump-deps/)

Specify library name and version:  **sdl/***

---

- [ ] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [ ] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/sdl/all/conanfile.py [hunk 1]
```diff
@@ -184,9 +184,9 @@ def build_requirements(self):
             # set. This could be because you are using a Mac OS X version less than 10.5
             # or because CMake's platform configuration is corrupt.
             # FIXME: Remove once CMake on macOS/M1 CI runners is upgraded.
-            self.tool_requires("cmake/3.25.3")
+            self.tool_requires("cmake/3.27.7")
```
Comment (jwillikers) [inline recipes/sdl/all/conanfile.py:187]: Should this be a version range?

**Review by jwillikers — CHANGES_REQUESTED** [review]

**Comment by ericLemanissier** [comment]
The build is now finished. You can take a look at the results [here](https://github.com/ericLemanissier/proof-of-conan/actions/runs/6533636028)

File: recipes/sdl/all/conanfile.py (hunk 1 above)
Comment (RubenRBS) [reply recipes/sdl/all/conanfile.py:187]: Yup

**Comment by ericLemanissier** [comment]
I don't understand this error, don't have time to work on it

**Pull request closed without merging**